### ISSUE TITLE
Notifications and sleep mode  😰

### ISSUE THREAD

**Issue by marktopper** [gitscout/gitscout-feedback#159]
I often get this when I open up app after my computer have been in sleep mode.

> Error: TypeError: Failed to fetch
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/workers.js:6090:23
From previous event:
    at loop (file:///Applications/Gitscout.app/Contents/Resources/app.asar/workers.js:5926:31)

**Comment by thibmaek** [comment]
Same here, every issue in my repo can't display content because of this error:

```console
TypeError: Cannot read property 'get' of undefined
    at _callee$ (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90288:75)
    at tryCatch (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:7449:41)
    at GeneratorFunctionPrototype.invoke [as _invoke] (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:7740:23)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:7501:22)
    at step (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90145:203)
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90145:384
    at tryCatcher (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37419:24)
    at Promise._settlePromiseFromHandler (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37134:32)
    at Promise._settlePromise (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37191:19)
    at Promise._settlePromise0 (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37236:11)
    at Promise._settlePromises (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37315:19)
    at Async._drainQueue (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:38007:17)
    at Async._drainQueues (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:38017:11)
    at Async.drainQueues (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37891:15)
From previous event:
    at Promise.longStackTracesCaptureStackTrace [as _captureStackTrace] (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:39063:20)
    at Promise._resolveFromExecutor (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:37102:11)
    at new Promise (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:36701:15)
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90145:100
    at getIssueActivityJson (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90521:18)
    at getIssueActivity (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90530:11)
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:90064:45
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:63192:19
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:30755:13
    at Container.fetchActivity (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:164473:8)
    at Container.componentDidMount (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:164371:13)
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:23072:26
    at measureLifeCyclePerf (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:22882:13)
    at file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:23071:12
    at CallbackQueue.notifyAll (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15508:23)
    at ReactReconcileTransaction.close (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:26223:27)
    at ReactReconcileTransaction.closeAll (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16687:26)
    at ReactReconcileTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16634:17)
    at ReactUpdatesFlushTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16621:21)
    at ReactUpdatesFlushTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15266:33)
    at flushBatchedUpdates (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15349:20)
    at ReactUpdatesFlushTransaction.close (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15224:8)
    at ReactUpdatesFlushTransaction.closeAll (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16687:26)
    at ReactUpdatesFlushTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16634:17)
    at ReactUpdatesFlushTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15266:33)
    at Object.flushBatchedUpdates (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15349:20)
    at ReactDefaultBatchingStrategyTransaction.closeAll (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16687:26)
    at ReactDefaultBatchingStrategyTransaction.perform (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:16634:17)
    at Object.batchedUpdates (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:25805:27)
    at Object.batchedUpdates (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:15274:28)
    at dispatchEvent (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:25962:21)
    at HTMLDocument.wrapped (file:///Applications/Gitscout.app/Contents/Resources/app.asar/rendered.js:60387:30)
```

**Comment by michael-lefebvre** [comment]
@marktopper and @thibmaek see #146, can you share public ticket/repo with this error please?

**Comment by michael-lefebvre** [comment]
@marktopper sorry, I read your message too fast. I see the problem source and it will be fix in next release.

I took the liberty to edit this ticket's title for further references

**Comment by michael-lefebvre** [comment]
@marktopper this must be fixed with v0.6.8, please confirm 🙂

**Comment by thibmaek** [comment]
Works
